tests/PHPUnit/Integration/Columns/DimensionTest.php

```diff
@@ -447,4 +447,40 @@ public function testRemovedDimensionNoLongerExists()
             $this->assertFalse(class_exists($removedDimension), "Dimension marked as removed but still exist: $removedDimension");
         }
     }
+
+    public function test_groupValue()
+    {
+        $this->dimension->setType(Dimension::TYPE_DURATION_MS);
```

Comment:
@snake14 Could you also add a testcase for `8883864` the one which highlighted the issue and also for some German locale values ?

Reply:
@AltamashShaikh I figured that [line 466](https://github.com/matomo-org/matomo/pull/20262/files#diff-41014f455afde9a2ad64a6ef22488b868ae2fef065fac5f164e356fa39fc4226R466) was similar enough to `8883864` that I didn't need to specifically add it. My current test uses a large enough number that it reproduced the error until I applied the fix. I don't think that the locale should matter since floats don't use local and I force the decimal and thousands separators for the `number_format` function.

Reply:
okay :+1: